CHANGELOG.md

```diff
@@ -19,6 +19,8 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
   [cyberark/conjur#151](https://github.com/cyberark/conjur/issues/151)
 - [Documentation](./UPGRADING.md) explaining how to upgrade a Conjur server deployed in a
   Docker Compose environment. [cyberark/conjur#1528](https://github.com/cyberark/conjur/issues/1528), [cyberark/conjur#1584](https://github.com/cyberark/conjur/issues/1584)
+- Add ability to specify cert installation path on `/authn-k8s/:service_id/inject_client_cert`.
+  [cyberark/conjur#1802](https://github.com/cyberark/conjur/pull/1802)
```

Comment:
this should be linked to the issue that was filed with this request, both here and in the PR description. can you please update the changelog entry and PR description to include this link (and drop the PR link here), and also add some text to the PR description to explain what this PR does?